src/config/config_type.rs

```diff
@@ -317,14 +340,17 @@ macro_rules! create_config {
                             }
                             name_out.push_str(name_raw);
                             name_out.push(' ');
-                            let mut default_str = format!("{}", $def);
+                            let default = <$ty as StyleEditionDefault>::style_edition_default(
```

Comment:
probably best to name this as something other than `default` given keyword overlap